protocol-definitions/custom/Custom.yaml

```diff
@@ -868,5 +868,49 @@ customTypes:
           type: Data
           nullable: true
           since: 2.0
-
-
+  - name: SqlQueryId
+    since: 2.1
+    params:
+      - name: memberIdHigh
+        type: long
+        nullable: false
+        since: 2.1
+      - name: memberIdLow
+        type: long
+        nullable: true
```

Comment:
is this nullable?

Reply:
Fixed (copy-paste artifact). It seems that this flag doesn't affect the generated code anyhow, so perhaps it could be made optional for primitive types?